scipy/stats/_distribution_infrastructure.py

```diff
@@ -3618,57 +3618,38 @@ class or its methods for more information.
 
     Create a custom distribution with variable support.
 
-    >>> class MyGenExtreme:
+    >>> class MyUniformCube:
     ...     @property
     ...     def __make_distribution_version__(self):
     ...         return "1.16.0"
     ...
     ...     @property
     ...     def parameters(self):
-    ...         return {"c": {}, "mu": {}, "sigma": (0, np.inf)}
+    ...         return {"a": (-np.inf, np.inf), "b": (-np.inf, np.inf)}
     ...
     ...     @property
     ...     def support(self):
-    ...         def left(*, c, mu, sigma):
-    ...             c, mu, sigma = np.broadcast_arrays(c, mu, sigma)
-    ...             result = np.empty_like(c)
-    ...             result[c >= 0] = -np.inf
-    ...             result[c < 0] = mu[c < 0] + sigma[c < 0] / c[c < 0]
-    ...             return result[()]
+    ...         def left(*, a, b):
+    ...             a = np.asarray(a)
+    ...             return (a**3)[()]
```

Comment:
```suggestion
    ...         def left(*, a, b):
    ...             return (a**3)
```
Are you sure this is needed? I mean, did you add it instinctively, or did you find a problem without it?
Testing locally, `a` seems to get an array if the class initializer parameter is array-like and a NumPy scalar if the class initializer parameter is a scalar.

Reply:
I just added it instinctively. I'll test if its needed.